guidelines/sc/22/visible-controls.html

```diff
@@ -5,12 +5,12 @@ <h4>Visible Controls</h4>
     <p class="conformance-level">AA</p>
 	<p class="change">New</p>
     
-    <p>Where receiving pointer hover or keyboard focus triggers <a>user interface components</a> to be visible, information needed to identify that user interface components are available is visible, except when:</p>
+    <p>When <a>user interface components</a> are invisible until hover or focus makes them visible, provide a visible indicator that the components are available. Except when:</p>
```

Comment:
Not sure starting a new sentence just with `Except when:` works that well. I'd stick with `, except when:`